fairlearn/reductions/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from fairlearn.reductions.grid_search_classification import GridSearchClassification  # noqa: F401
```

Comment:
what about Regression?

Reply:
I think that will be a separate class - GridSearchRegression. I wanted this reviewed first, to make sure I'm not doing anything which no one was expecting.

Reply:
Have reworked as we agreed in the meeting.